pages/common/git-local-commits.md

```diff
@@ -0,0 +1,9 @@
+# git local-commits
+
+> Show local commits which haven't been pushed to origin. Any additional arguments will be passed directly to `git log`.
+> Part of `git-extras.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-local-commits>.
+
+- Show commits haven't been pushed:
```

Comment:
```suggestion
- Show commits that have not been pushed:
```

Reply:
What about 
```suggestion
- Show commits that haven't been pushed:
```

Reply:
Either are fine with me.  When and when not to use contractions can be tricky 🤷🏻‍♂️

Reply:
Having the contraction makes more sense in this case. I have commited it